tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
 envlist =
        {py38,py39}-{django42}
-       {py310}-{django42,django50,django51,djangomain}
-       {py311}-{django42,django50,django51,djangomain}
-       {py312}-{django42,django50,django51,djangomain}
-       {py313}-{django51,djangomain}
```

Comment:
If this changes removing 5.0 from the test matrix, shouldn't it be removed from the `docs/index.md` too?

Reply:
It is not necessary